doc/changes.rst

```diff
@@ -1,6 +1,111 @@
 Changelog
 =========
 
+Release v0.3.0 (2021/02/15)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4522136.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.4522136
+
+Highlights
+
+*
```

Comment:
We'll need to add 3 to 4 lines for the highlights section here. How about:

```suggestion
* 🎉 **Third minor release of PyGMT** 🎉
* Wrap inset (#788) for making overview maps and subplot (#822) for multi-panel figures
* Apply standardized formatting conventions (#775) across most documentation pages
* Drop Python 3.6 support (#699) so PyGMT now requires Python 3.7 or newer
```
Feel free to suggest edits in the wording. I'm also thinking whether to mention the upcoming drop in support for GMT 6.1.1 (once GMT 6.2.0 is out, something for PyGMT v0.4.0) and/or NEP29 adoption (#690). Might depend on whether we want a v0.3.1 release?

Reply:
> I'm also thinking whether to mention the upcoming drop in support for GMT 6.1.1 (once GMT 6.2.0 is out, something for PyGMT v0.4.0) and/or NEP29 adoption

Don't try to predict the future.

Reply:
Ok, will surprise our users then :smile_cat: